### ISSUE TITLE
Compiler Error:  error: extra qualification ‘BirdCurveHelper::’ on member ‘micro2bary’ [-fpermissive]

### ISSUE THREAD

**Issue by tyler274** [nvpro-samples/vk_mini_samples#3]
Attempting to build on both Ubuntu and NixOS results in this error once dependencies have been sorted, on both GCC and Clang

```
[ 72%] Building CXX object samples/mm_displacement/CMakeFiles/mm_displacement.dir/src/dmm_process.cpp.o
In file included from /home/tport/code/vulkan_samples/vk_samples/samples/mm_displacement/src/dmm_process.cpp:34:
/home/tport/code/vulkan_samples/vk_samples/common/bird_curve_helper.hpp:66:15: error: extra qualification ‘BirdCurveHelper::’ on member ‘micro2bary’ [-fpermissive]
   66 |   static void BirdCurveHelper::micro2bary(uint32_t index, uint32_t subdivisionLevel, nvmath::vec3f& uv0, nvmath::vec3f& uv1, nvmath::vec3f& uv2);
      |               ^~~~~~~~~~~~~~~
gmake[2]: *** [samples/mm_displacement/CMakeFiles/mm_displacement.dir/build.make:109: samples/mm_displacement/CMakeFiles/mm_displacement.dir/src/dmm_process.cpp.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:781: samples/mm_displacement/CMakeFiles/mm_displacement.dir/all] Error 2
gmake: *** [Makefile:136: all] Error 2
```

There's other compile time errors I'm still sorting out around the `equal` and `hash` lambdas....

**Comment by tyler274** [comment]
titular issue fixed in https://github.com/nvpro-samples/vk_mini_samples/pull/4

**Comment by mklefrancois** [comment]
Thanks for reporting, this error should be fixed in b38052d.
Closing.
